src/libs/Notification/PushNotification/index.native.js

```diff
@@ -70,6 +70,12 @@ function init() {
     UrbanAirship.addListener(EventType.NotificationResponse, (event) => {
         pushNotificationEventCallback(EventType.NotificationResponse, event.notification);
     });
+
```

Comment:
Just one thought, could you add a comment explaining what this configuration is for

Reply:
+1. Also note that this is only for iOS in that comment

Reply:
great idea. Added. Let me know if it is suggestive enough.